vrs/marten-construction-script.py

```diff
@@ -0,0 +1,364 @@
+    # Feel free to uncomment and run to check the values here for yourself if you would like 
+    """
+
+    # Frequencies of the variant types in gnomAD v3.1.2
+    # This may need to be changed with future releases or versions
+    indel_freq = 0.14525709561723196
+    ref_freq = 0.02122722096390106
+    var_freq = 0.0381956465303165
+    x_freq = 0.0398928560027597
+    y_freq = 0.0015374206699161612
+    random_freq = 1.00
+    whole_count = 759302267
+    # e.g.: For 1000 variants, 145 will be indels
+    # for 1000 variants, 40 will be on chromosome X and only 1.5 will be on chromosome Y
+
+    # Downsample the matrix table if the User doesn't have the capacity to annotate and go through 759million variants
+    if args.downsample is float and args.downsample < 1.00:
+        ht_whole = ht_whole.sample(args.downsample)
+        whole_count *= args.downsample
+
+    # Construct Tables for each of the targeted variant types desired for the test subset:
+    # DEFAULT: 50k random, 10k extra indels, 10k additional long references, 10k  additional long variants, 10k on sex chromosomes (9k on X, 1k on Y)
+
+    # Create freq dictionary with variant type as the key and pre-calculated frequencies as the value
+    freq_dict = {
+        "indel": indel_freq,
+        "long_ref": ref_freq,
+        "long_var": var_freq,
+        "on_x": x_freq,
+        "on_y": y_freq,
+        "random": random_freq,
+    }
+
+    # Create desired counts dictionary with variant type as the key and desired number of that variant type in the test subset as the value
+    # NOTE: would like to incorporate Multiallelic and Min-Rep into this dictionary (and above)
+    # BUT since they are generated very differently, it's difficult to work out.
+    desired_counts = {
+        "random": 50000,
+        "indel": 10000,
+        "long_ref": 10000,
+        "long_var": 10000,
+        "on_x": 9000,
+        "on_y": 1000,
+    }
+
+    # Reset the desired counts if the user chooses to change them - Multiallelic stored at [-2], Min-Rep at [-1]
+    counts_list = []
+    if args.custom:
+        counts_list = []
+        if args.custom:
+            argument_list = args.counts.split(",")
+            for argo in argument_list:
+                counts_list.append(argo)
+
+        for obj in enumerate(desired_counts.keys()):
+            desired_counts[obj[1]] = int(counts_list[obj[0]])
+
+    logger.info(f"Counts as: {desired_counts}")
+
+    # Annotate the Table with the status of Allele we are looking at
+    ht_whole = ht_whole.annotate(
+        indel=hl.is_indel(ht_whole.alleles[0], ht_whole.alleles[1]),
+        long_ref=hl.len(ht_whole.alleles[0]) > 5,
+        long_var=hl.len(ht_whole.alleles[1]) > 5,
+        on_x=(ht_whole.locus.contig == "chrX"),
+        on_y=(ht_whole.locus.contig == "chrY"),
+        random=(True),
```

Comment:
```suggestion
        random=True,
```
you shouldn't need the parentheses here